### DISCUSSION TITLE
What's this after checkout branch `0.10`?

### DISCUSSION THREAD

**Discussion by yinghaoyu** [ayamir/nvimdots#1312] [Q&A] [answered]
```c
int f(int x, int y)
{
  return std::min(a: x, b: y);
}
```

`a:` and `b:` is not code  section, how can i remove them?

**Comment by charliie-dev** [comment]
it's snippets. u can fill your desired func name and variable as u need.

**Reply by yinghaoyu** [reply]
`std::min(x, y)` is enough, `a:` and `b:` mess up in vision.

**Reply by charliie-dev** [reply]
u can create your own snippets, it's the default one.

**Reply by yinghaoyu** [reply]
![Snipaste_2024-06-28_15-13-11](https://github.com/ayamir/nvimdots/assets/42765754/8ce97397-7a98-4dac-b5eb-adf2a006dc6a)

**Reply by yinghaoyu** [reply]
I think it's not a snippet? Did you misunderstand？

**Reply by yinghaoyu** [reply]
I had already finished coding.

**Reply by charliie-dev** [reply, marked as answer]
https://github.com/ayamir/nvimdots/blob/57f647e9d302ed03bc80edc676be7c49f1f986ad/lua/core/settings.lua#L13

it's called `inlay_hists`, u can set it to `false`.

**Reply by yinghaoyu** [reply]
Thanks.
